evap/staff/templates/staff_import_participants.html

```diff
@@ -0,0 +1,21 @@
+{% extends "staff_semester_base.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block content %}
+    {{ block.super }}
+    <p>
+        {% trans "Upload course participants data from excel file (" %}<a href="{% get_static_prefix %}sample_user.xls">{% trans "Sample File"%}</a>{% trans "). This will create all containing students and enroll them into the course: " %} {{ course }}
```

Comment:
Please use `This will create user accounts for all contained students and adds them as participants for the course`